com.unity.render-pipelines.universal/Editor/AssetPostProcessors/MaterialPostprocessor.cs

```diff
@@ -38,6 +57,11 @@ static void ReimportAllMaterials()
                 materialIdx++;
                 var path = AssetDatabase.GUIDToAssetPath(asset);
                 EditorUtility.DisplayProgressBar("Material Upgrader re-import", string.Format("({0} of {1}) {2}", materialIdx, totalMaterials, path), (float)materialIdx / (float)totalMaterials);
+                var material = (Material)AssetDatabase.LoadAssetAtPath(asset, typeof(Material));
```

Comment:
Sadly this will slow up the import :/
Not sure how we can prevent it.

Reply:
@bastienunity do you have an idea on how i can properly enforce to process Materials from only the RenderPipeline I am interested in?

Reply:
I don't think you can as you have to load them in memory to figure out the shader.
However, there are probably some significant improvements that could be made in this method:
- Don't use AssetDatabase.FindAssets("t:material", null); but AssetDatabase.FindAssets("glob:\"*.mat\""); instead. This will find guids only for .mat assets and will avoid loading in memory materials that you are not supposed to change.
- What is AssetDatabase.ImportAsset doing? You probably want to do an AssetDatabase.SaveAssetIfDirty instead?
- If Importers create Materials using a Shader that needs to be updated based on the SRP installed/active, then they need to declare a dependency upon it. We can discuss that separately on what is expected and how we can implement it. In any case, it's not viable to modify them after the import like that. They are never being saved back anywhere, and when unloaded they may be reverted to before your change (especially given the ModelImporter is always using a local caching now)

Reply:
Thanks! 

- I implemented the glob search suggestion,

- i think we are retriggering import to ensure the Material Upgrader is run. I dont think SaveAsset will trigger this behavior. no?
- i guess this point is out of scope for this bugfix but yes lets discuss it - do you agree?

Reply:
note: Bastien and I had a discussion there are some things I could implement for this PR.
cc @phi-lira